routes/user.js

```diff
@@ -24,4 +24,8 @@ router.post('/projects/create', async (req, res) => {
     } 
 })
 
+router.get('/team', async (req, res) => {
```

Comment:
May need to create additional route paths `/teams` or `/specific_team_id` depending on if the user is looking at all their teams or just one team.